recipes/pymannkendall/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "pymannkendall" %}
+{% set version = "1.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a10a1cc24c7466bce7e1c0c70c5601ad44aee573d2ce161e6e68b60f6a94e11e
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+
```

Comment:
This can be `noarch`, right?

Reply:
Yes, I always forget that the skeleton doesn't do that.